src/intersection.ts

```diff
@@ -112,16 +112,19 @@ export function unobserve(element: Element | null) {
     let rootObserved = false
     /* istanbul ignore else  */
     if (observerId) {
-      INSTANCE_MAP.forEach((item, key) => {
-        if (item && key !== element) {
-          /* istanbul ignore else  */
-          if (item.observerId === observerId) itemsLeft = true
-          /* istanbul ignore else  */
-          if (item.observer.root === root) rootObserved = true
+      for (let [key, item] of INSTANCE_MAP) {
```

Comment:
It looks like this line gets compiled to
```js
for (var _iterator = INSTANCE_MAP, _isArray = Array.isArray(_iterator), _i = 0, _iterator = _isArray ? _iterator : _iterator[Symbol.iterator]();;) {
```
which breaks IE10 and IE11 because `Symbol` is undefined.

Reply:
This is why we can't have nice things. 😩
Thanks for the heads-up. Guess it needs to be rewritten

Reply:
That said, you should be able to polyfill `Symbol` using babel-polyfill.

Reply:
Thanks for the quick fix!